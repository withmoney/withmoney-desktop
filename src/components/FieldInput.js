import React from 'react';

const FieldInput = ({ ...othersProps }) => (
  <div className="field">
    <input
      type="text"
      className="field__input"
      {...othersProps}
    />
  </div>
);

export default FieldInput;
