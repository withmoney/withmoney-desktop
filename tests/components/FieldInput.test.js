import renderer from 'react-test-renderer';
import FieldInput from 'components/FieldInput';

describe('FieldInput', () => {
  let Component;

  beforeAll(() => {
    Component = global.setup(FieldInput);
  });

  it('should render a component correctly', () => {
    const wrapper = renderer.create(
      Component({
        id: 'input-id',
      }),
    );

    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
