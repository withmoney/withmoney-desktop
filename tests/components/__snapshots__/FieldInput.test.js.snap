// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FieldInput should render a component correctly 1`] = `
<div
  className="field"
>
  <input
    className="field__input"
    id="input-id"
    name="input-id"
    type="text"
  />
</div>
`;
